### PR TITLE
If applied this commit will print out twenty hello messages

### PR DESCRIPTION
**Pull request by eggsy84**
Hey @eggsy84 

Can you take a look at this

### CODE REVIEW

File: src/main/java/Application.java [hunk 1]
```diff
@@ -4,5 +4,9 @@ public static void main (String [] args) {
         System.out.println("Hi there, How are you?");
         System.out.println("Hello team");
         System.out.println("There we go");
+
+        for (int i =0; i < 20; i++) {
+            System.out.println("Hello");
+        }
```
Comment (eggsy84) [inline src/main/java/Application.java:10]: Hey looks great!

Comment (MayaVs) [reply src/main/java/Application.java:10]: I can reply here

Comment (neelamdev) [reply src/main/java/Application.java:10]: can comment

Comment (papatekken) [reply src/main/java/Application.java:10]: comment from Allan

Comment (abinormal) [reply src/main/java/Application.java:10]: Love your for loop, shiny!

**Comment by SudhaLV** [comment]
Reviewed :) !!

**Pull request merged**